### DISCUSSION TITLE
[REQ]

### DISCUSSION THREAD

**Discussion by 98kx** [Chia-Network/chia-blockchain#3189] [General]
How to build windows chia.exe by build_windows.ps1 ?
It seems not working .

**Comment by 98kx** [comment]
Invoke-WebRequest -Uri "https://pypi.chia.net/simple/miniupnpc/miniupnpc-2.1-cp37-cp37m-win_amd64.whl" -OutFile "miniupnpc-2.1-cp37-cp37m-win_amd64.whl"
Write-Output "Using win_amd64 python 3.7 wheel from https://github.com/miniupnp/miniupnp/pull/475 (2.2.0-RC1)"


I can't install miniupnpc  to windows .
ERROR: miniupnpc-2.1-cp37-cp37m-win_amd64.whl is not a supported wheel on this platform.

**Reply by ntheile** [reply]
i had to downgrade Python from 3.9 to 3.7. You can check the version of python installed by running `python --version`
